### DISCUSSION TITLE
How do I activate AI skeleton recognition through C# SDK?

### DISCUSSION THREAD

**Discussion by abiliaorg** [3DiVi/nuitrack-sdk#67] [General]
In the last version one has the possibility to recognize users using an AI model. How do I activate this feature through code? I'm working with C# SDK. Thanks!

**Comment by VadimAspirin** [comment]
Hi @abiliaorg, 

You need to add this string after `Nuitrack.Init` to enable AI Skeleton Tracking through code:

```cs
Nuitrack.SetConfigValue("Skeletonization.Type", "CNN_HPE");
```

**Reply by VadimAspirin** [reply]
Hi @abiliaorg,

How are you? Has your issue been solved? 
Would be great if you could provide some reply/feedback, we will be ready to help.
Thanks.
